server/tserver/src/main/java/org/apache/accumulo/tserver/scan/LookupTask.java

```diff
@@ -76,6 +77,11 @@ public void run() {
       TableConfiguration acuTableConf = server.getTableConfiguration(session.threadPoolExtent);
       long maxResultsSize = acuTableConf.getAsBytes(Property.TABLE_SCAN_MAXMEM);
 
+      // Always return large root tablets
+      if (session.threadPoolExtent.tableId().equals(AccumuloTable.ROOT.tableId())) {
+        maxResultsSize = Long.MAX_VALUE;
```

Comment:
The property default is 512K, so maybe Long.MAX_VALUE is not the right choice here

Reply:
I agree that going smaller than Long.MAX_VALUE makes sense.

Maybe the best thing is to just take the maximum value between the default and the user setting so we always use at least the default size but allow users to go higher.

```java
maxResultsSize = Math.max(maxResultsSize, ConfigurationTypeHelper
  getFixedMemoryAsBytes(Property.TABLE_SCAN_MAXMEM.getDefaultValue()));
```

Reply:
Implemented in 13249fb